development.cfg

```diff
@@ -12,3 +12,26 @@ zcml-additional-fragments +=
     <class class="plone.app.blob.content.ATBlob">
         <implements interface="ftw.labels.interfaces.ILabelSupport" />
     </class>
+
+[instance]
+eggs +=
+    ${debug:eggs}
+
+zcml +=
+    ${debug:zcml}
+
+[versions]
+setuptools =
+zc.buildout =
+
+[debug]
```

Comment:
Since those are personal preferences, I would suggest that you move those into your own `buildout.cfg` locally. We have some standards at 4teamwork for debugging (e.g. we use `pdb`) which we would like to keep consistently across all of our packages.